packages/token/src/tests/mocks/erc1155_receiver_mocks.cairo

```diff
@@ -1,4 +1,4 @@
-use openzeppelin_utils::test_utils::constants::SUCCESS;
+const SUCCESS: felt252 = 123123;
```

Comment:
As mentioned previously, I suggest changing this to the short string `'SUCCESS'`